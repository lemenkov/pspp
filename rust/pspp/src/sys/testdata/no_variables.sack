# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; # Layout code
0; # Nominal case size (empty)
0; # Not compressed
0; # Not weighted
0; # 0 cases.
100.0; # Bias.
"01 Jan 11"; "20:53:52"; s64 "PSPP synthetic test file";
i8 0 *3;

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# Dictionary termination record.
999; 0;
