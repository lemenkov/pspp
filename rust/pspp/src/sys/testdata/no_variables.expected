╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │None                    │
│       Number of Cases│                       0│
╰──────────────────────┴────────────────────────╯

╭─────────┬────────────────────────╮
│Label    │PSPP synthetic test file│
│Variables│                       0│
╰─────────┴────────────────────────╯

                                            Variables
